<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QDialog" name="setting">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1. ÏúÑÌóòÍµ¨Ïó≠ ÏßÑÏûÖ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>54</width>
     <height>12</height>
    </size>
   </property>
   <property name="text">
    <string>ÏïåÎ¶º ÏÑ§Ï†ï</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>UI ÌÖçÏä§Ìä∏ ÍµµÍ∏∞ ÏÑ§Ï†ï</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2. ÏÇ¨Îûå - ÏßÄÍ≤åÏ∞® Í∞Ñ Ï∂©Îèå Í±∞Î¶¨ ÏßÑÏûÖ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3. </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ÎØ∏Î¶¨Î≥¥Í∏∞ ÌÖçÏä§Ìä∏</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15px</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20px</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ÌååÏùº Ï†ÄÏû• ÏúÑÏπò Î≥ÄÍ≤Ω</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ÌòÑÏû¨ :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>C:KDTvideo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="video_before_button_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>51</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>üìÇ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
